Package.swift

```diff
@@ -19,10 +19,10 @@ let package = Package(
     name: "swift-nio-http2",
     products: [
         .executable(name: "NIOHTTP2Server", targets: ["NIOHTTP2Server"]),
-        .library(name: "NIOHTTP2", targets: ["NIOHTTP2"])
+        .library(name: "NIOHTTP2", targets: ["NIOHTTP2"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-nio.git", from: "1.7.0"),
+        .package(url: "https://github.com/apple/swift-nio.git", .branch("master")),
```

Comment:
None of the tagged builds of swift-nio contain the fix for the precondition in `ByteBufferView.init()`, and that was causing me to crash all over the place when taking a view of the last chunk of a buffer 😉 If there's been a version-tagged release since last week which includes that change, I'll be happy to take that, because as I'm sure you're aware, targeting `master` isn't a *great* solution…

Reply:
Just checked, and the v1.8.0 tag on swift-nio still contains the following broken precondition in `ByteBuffer-views.swift`:

```swift
precondition(range.lowerBound >= 0 && range.upperBound < buffer.capacity)
```

Reply:
Yup, there isn't a release yet. 1.9.0 is probably a few weeks away at the moment, so we just need to remember this is here ;)